src/components/teacher-feedback/activity-level-feedback-banner.tsx

```diff
@@ -9,11 +11,25 @@ interface IProps {
 }
 
 export const ActivityLevelFeedbackBanner = ({ teacherFeedback }: IProps) => {
+  const hasRubric = !!teacherFeedback.feedbackSettings?.rubric;
+  const bannerClass = classNames("activity-level-feedback-banner", {
+    "has-rubric": hasRubric
+  });
+
   return (
-    <div className="activity-level-feedback-banner" data-testid="activity-level-feedback-banner">
+    <div className={bannerClass} data-testid="activity-level-feedback-banner">
       <TeacherFeedbackIcon className="teacher-feedback-icon" />
+      {hasRubric && (
```

Comment:
Should the text in this section be wrapped in the `<DynamicText>` component?

Reply:
Yes, thanks. I've added it here and in the other places you noted.